hangman.rb

```diff
@@ -0,0 +1,195 @@
+require 'colorize'
+
+class Hangman
+	def initialize
+		@words = ["method", "index", "bracket", "braces", "gem", "git", "terminal", "length", 
```

Comment:
I don't think you knew this at the time, but you could make the `@words` instance variable a constant since you don't modify this as you go along.